### PR TITLE
dev: slightly decrease the rate at which we send transaction

### PR DESCRIPTION
**Pull request by bajpai244**
In the current benchmarking CI, ofthen there occurs a nonce race, and from my experience, the way to solve it is to decrease the rate at which send the transaction, this PR slightly bumps the delay between two transactions by 150ms, which should probably resolve it, otherwise we would have to move to a custom runner.

# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [x] Build-related changes
- [ ] Documentation content changes
- [ ] Testing

# What is the new behavior?

- Decreases the rate at which we send transactions which should lower the chance of getting a nonce race.

# Does this introduce a breaking change?

- [ ] Yes
- [x] No

# Added changes to [`CHANGELOG.md`](../CHANGELOG.md)

- [x] Yes
- [ ] No

### CODE REVIEW

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     
     on:
       push:
-        branches: [main]
+        branches: [main, dev/update-benchmark-script]
```
Comment (Eikix) [inline .github/workflows/benchmark.yml:6]: is this still relevant

File: benchmarking/scripts/processor.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ const nativeTokenTransfer = async (context, ee, next) => {
 
     // break to make sure transactions arrive in order to RPC.
     // NOTE: reduce it if you are skipping validate
-    await sleep(350);
+    await sleep(500);
```
Comment (Eikix) [inline benchmarking/scripts/processor.js:40]: why 500? could this slow tps if 500>time to post tx

**Review by Eikix — COMMENTED** [review]
questions:) but lgtm

File: .github/workflows/benchmark.yml (hunk 1 above)
Comment (bajpai244) [reply .github/workflows/benchmark.yml]: nope, just added it and have removed it as well, was for local fork testing!

File: benchmarking/scripts/processor.js (hunk 2 above)
Comment (bajpai244) [reply benchmarking/scripts/processor.js:40]: So madara block time is 6second.

So, if means atleast with this we will send 12 transactions per block. So, if our TPS is less than let's say even 3, that means the pool has enough transactions, and they are not being processed!

Also, the script waits for some time for pool to be filled with transactions before collection any data.

**Comment by bajpai244** [comment]
> questions:) but lgtm

have replied :)

**Review by Eikix — DISMISSED** [review]
lgtm

**Review by Eikix — APPROVED** [review]
lgtm

**Pull request merged**